src/plugins/ab-testing-service.ts

```diff
@@ -62,6 +62,14 @@ function getEnvironment() {
   //   ABTestingEnvironment.name_of_the_test.value ||
   //   (Math.random() > 0.5 ? "A version name" : "B version name")
 
+  const versions = ["version_actuelle", "version_test_1", "version_test_2"]
+  ABTestingEnvironment.CTA_EmailRecontact =
+    ABTestingEnvironment.CTA_EmailRecontact || {}
+  ABTestingEnvironment.CTA_EmailRecontact.index = 5
+  ABTestingEnvironment.CTA_EmailRecontact.value =
+    ABTestingEnvironment.CTA_EmailRecontact.value ||
+    versions[Math.floor(Math.random() * versions.length)]
```

Comment:
J'crois qu'on peut encore simplifier tes modif (tu me dis si je suis relou ^^)
```suggestion
const ctaEmailRecontact = ABTestingEnvironment.CTA_EmailRecontact || {};
ctaEmailRecontact.index = 5;
ctaEmailRecontact.value = ctaEmailRecontact.value || versions[Math.floor(Math.random() * versions.length)];
```

Reply:
Haha tkt :-)
Ouais ok mais avec l'assignation en plus de `ctaEmailRecontact` à 1ABTestingEnvironment.CTA_EmailRecontact` du coup